tests/unit/resources/test_factory.py

```diff
@@ -188,13 +233,25 @@ def test_dangling_resource_create_with_kwarg(self):
             }
         }
 
-        resource = self.load('test', 'test', {}, defs, None)()
+        resource = self.load('test', 'test', model, defs, None)()
         q = resource.Queue(url='test')
 
         self.assertIsInstance(q, ServiceResource,
             'Dangling resource created with kwargs is not a ServiceResource')
 
     def test_dangling_resource_shares_client(self):
+        model = {
```

Comment:
I'm inclined to agree here.  My main issue with tests is being able to look at the test and understand what's being tested.  I'd either suggest:

a) what Kyle said, but break this out into it's separate test class where only the tests that have this shared model are in this test class.

b) cut down the boiler plate with a builder (my personal preference).  It seems like the only interesting bit here is a resource with identifier X, and what that means for instantiating.  e.g. resource with source input and target X can i) be instantiated with positions args ii) will raise a value error is instantiated with X and Y, etc.

In fact, in this test here it looks like the test is that the resource and the queue share the client, in which case writing out the model in the test makes it harder to read, because it doesn't actually matter what the model is, we're checking the `client` property on the test.